.setup/bin/setup_sample_courses.py

```diff
@@ -61,6 +61,9 @@ def main():
     """
     global DB_ONLY
 
+    # To make Iris testing possible, need to seed random
+    random.seed(10090542)
```

Comment:
👍 

Yeah, I'm fine with the seed changing if we change the code the course as we're at that point totally redesigning the course. That is probably the most stable of the things about a course though (after we do #1075 of course).